.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,53 @@
+name: phpunit tests
+
+on:
+  push:
```

Comment:
It might be worth making this into a reusable workflow called by the Workflow PR to allow the PR workflow to fail fast if unit tests fail

Reply:
Thank you, that is the ultimate plan :). note this PR is Draft. ;)